src/webdriver-plus.ts

```diff
@@ -18,7 +18,10 @@ export interface IFindInterface {
   /**
    * Shorthand to find all element matching a css selector.
```

Comment:
Please include a comment, e.g. like this "If mapper is provided, it will be applied to each of the found elements, and findAll() will return the resolved results. E.g. findAll('a', (el) => el.getAttribute('href'))"

Also, we can make typing more precise using [function overloads](https://www.typescriptlang.org/docs/handbook/functions.html#overloads), and I think we should, e.g.
```
findAll(selector: string): promise.Promise<WebElement[]>;
findAll<T>(selector: String, mapper: (e: WebElement) => Promise<T>): Promise<T[]>;
```

The actual implementation can stay as you have it.